### ISSUE TITLE
generated PAGE-XML: text with and without namespace?

### ISSUE THREAD

**Issue by jbarth-ubhd** [cisocrgroup/pocoweb#65]
within PAGE-XML within .zip I'll see such snippets:

```xml
  <pc:TextRegion id="region0005" orientation="-0.20483407817602" readingDirection="left-to-right" textLineOrder="top-to-bottom">
   <pc:Coords points="..."></pc:Coords>
   <pc:TextLine id="region0005_line0000">
    <pc:Coords points="..."></pc:Coords>
    <pc:TextEquiv conf="0.444073736667633">
     <pc:Unicode>e.</pc:Unicode>
    </pc:TextEquiv>
    <TextEquiv>
     <Unicode>ESSE QUAM VIDERI</Unicode>
    </TextEquiv>
    <Word>
     <TextEquiv>
      <Unicode></Unicode>
     </TextEquiv>
     <TextEquiv>
      <Unicode>ESSE</Unicode>
     </TextEquiv>
    </Word>
    <Word>
     <TextEquiv>
      <Unicode></Unicode>
     </TextEquiv>
     <TextEquiv>
      <Unicode>QUAM</Unicode>
     </TextEquiv>
    </Word>
    <Word>
     <TextEquiv>
      <Unicode></Unicode>
     </TextEquiv>
     <TextEquiv>
      <Unicode>VIDERI</Unicode>
     </TextEquiv>
    </Word>
   </pc:TextLine>
   <pc:TextEquiv>
    <pc:Unicode>e.</pc:Unicode>
   </pc:TextEquiv>
  </pc:TextRegion>
```

**Comment by finkf** [comment]
Thanks for reporting. I will look into it.

**Comment by finkf** [comment]
fixed in c53d0fc7
